scripts/vellum-to-hq

```diff
@@ -170,12 +173,10 @@ def update_hq(path, branch, base_branch, vellum_dir, vellum_rev, push, target, d
     if branch == 'vellum-staging':
         message += """
 
-        Oops. Looks like the vellum team forgot to update this branch
-        when updating master. To get past this you can run:
+        If you are running rebuildstaging and this commit conflictcs with
```

Comment:
It's not likely who is currently running `vellum-to-hq` is also running `rebuildstaging` at the same time, so this if statement might not applicable to the user. User might don't even realize there will be a conflicts until they run `rebuildstaging`. Can you update this message?

Reply:
This is part of a commit message on the `vellum-staging` branch, which should be seen by someone encountering conflicts between `vellum-staging` and `master`. This will probably be seen by a person running `rebuildstaging`, not necessarily when they are using `vellum-to-hq`. This message is intended to direct them to the easiest way to fix the conflict.